addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1429,7 +1429,10 @@ msgctxt "#306"
 msgid "Non-interleaved"
 msgstr ""
 
-#empty string with id 307
+#: xbmc/LangInfo.cpp
+msgctxt "#307"
+msgid "Media default stream"
```

Comment:
should we drop "stream" here and also from "Original stream's language"? I personally think less tech savvy users have no idea what it is, and for the more tech savvy users it doesn't make a difference if it's in the label or not (IMO). So I don't see a real benefit in having it as part of the label.
Also, please add a contextual note so that translators can provide a sane translation based on the context it's used, without having to browse the source code (which no translator does). So basically add something like
`#. Select option for the setting "Preferred audio language". Will prefer the stream marked as "default" if available.`